src/main/java/net/mcreator/generator/template/TemplateExpressionParser.java

```diff
@@ -27,13 +27,12 @@
 import javax.annotation.Nullable;
 import java.io.StringReader;
 import java.io.StringWriter;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
+import java.util.*;
+import java.util.concurrent.atomic.AtomicReference;
 
-public class TemplateConditionParser {
+public class TemplateExpressionParser {

```

Comment:
This refactor belongs in separate PR

Reply:
Before it was only used for conditions, now it is used to get lists as well, so keeping this named _**Condition** parser_ would be invalid from semantic point of view.

Reply:
Though I can split the `getValueFrom()` part actually, but would it maybe be good to move ALL the changes here to another pull?

Reply:
Yep, this could be another PR. Just keep what relates to this PR for now.

And in fact even now it has ftl expression parser and is called TemplateConditionParser

Reply:
#2756